hv_func.h

```diff
@@ -82,6 +86,12 @@
 */
 #include "perl_siphash.h"
 
+#define __PERL_HASH_SEED_roundup(x, y)   ( ( ( (x) + ( (y) - 1 ) ) / (y) ) * (y) )
```

Comment:
Shouldn't need to round if things are correctly sized.

Reply:
We derive the word size from the rounded up byte size so that  we can deal with mixed U64 and U32 data.